### PR TITLE
Refactor/model classes

### PR DESCRIPTION
**Pull request by lsantire**
## Changes proposal

- Deleted unused model classes
- Moved some model classes out of model.py

## Testing

Local manual testing

### CODE REVIEW

File: iot_api/iotservice.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from iot_api import app, api, jwt
 from iot_api.user_api import db
 from iot_api import mail, socketio
-from iot_api.user_api.model import RevokedTokenModel
+from iot_api.user_api.models.RevokedTokenModel import RevokedTokenModel
```
Comment (cyones) [inline iot_api/iotservice.py]: Cannot be imported directly from iot_api.user_api.models?

`from iot_api.user_api.models import RevokedTokenModel`

File: iot_api/user_api/resources/endpoints.py [hunk 2]
```diff
@@ -27,20 +27,26 @@
 from iot_api.user_api.enums import RoleTypes
 from iot_api.user_api.events.data_collector_events import emit_data_collector_event
 from iot_api.user_api.events.policy_events import emit_policy_event
-from iot_api.user_api.model import User, Organization, Device, RevokedTokenModel, AccountActivation, StatsCounters, \
-    PasswordReset, LoginAttempts, UserRole, UserToUserRole, ChangeEmailRequests, Alert, AlertType, Packet, \
-    SendMailAttempts, GlobalData, \
+from iot_api.user_api.model import User, Organization, Device, AccountActivation, \
+    UserRole, UserToUserRole, Alert, AlertType, Packet, \
     get_user_collector_ids
+from iot_api.user_api.models.ChangeEmailRequests import ChangeEmailRequests
```
Comment (cyones) [inline iot_api/user_api/resources/endpoints.py]: Same here, the models can be imported directly from models, since they are in the models __init__ file.

**Review by cyones — CHANGES_REQUESTED** [review]

File: iot_api/iotservice.py (hunk 1 above)
Comment (lsantire) [reply iot_api/iotservice.py]: You are right, done!

File: iot_api/user_api/resources/endpoints.py (hunk 2 above)
Comment (lsantire) [reply iot_api/user_api/resources/endpoints.py]: Done!

**Review by cyones — APPROVED** [review]

**Pull request merged**